pom.xml

```diff
@@ -259,20 +259,6 @@
                     <groupId>org.apache.maven.plugins</groupId>
                     <artifactId>maven-javadoc-plugin</artifactId>
                 </plugin>
-                <plugin>
-                    <groupId>org.mule.tools.maven</groupId>
-                    <artifactId>mule-module-maven-plugin</artifactId>
-                    <version>${muleModuleMavenPluginVersion}</version>
-                    <executions>
-                        <execution>
-                            <id>analyze</id>
-                            <phase>verify</phase>
-                            <goals>
-                                <goal>analyze</goal>
-                            </goals>
-                        </execution>
-                    </executions>
-                </plugin>
```

Comment:
I don't get what does this have to do with the subject of the PR.

Reply:
edited the title